storage/client/CMakeLists.txt

```diff
@@ -52,7 +54,7 @@ foreach (fname ${storage_client_unit_tests})
     set(target "storage_client_${target}")
     add_executable(${target} ${fname})
     target_link_libraries(${target} PRIVATE
-        storage_client gmock CURL::CURL storage_common_options)
+        storage_client gmock CURL::CURL storage_common_options nlohmann_json)
```

Comment:
Nit: (Here and in other places) IMO, it's much more readable to break these up into separate lines, similar to the target_include_directories command above (L. 26).  I was going to go down the rabbit hole of suggesting auto-formatting, but it doesn't look like a great CMake file formatter exists :(  Something like the these examples below is much easier to mentally parse, but at the expense of taking up lots more vertical whitespace... I'd personally prefer readability over fewer lines:

    target_link_libraries(target
        PRIVATE
            lib_a
            lib_b
        PUBLIC
            lib_c)

or


    target_link_libraries(target
        PRIVATE lib_a
                lib_b
        PUBLIC lib_c)

as these CMake files get larger and larger, I'd like to prevent them from becoming impossible to read; might as well start now.  Thoughts?

Reply:
I think your first alternative is the most readable, though it makes the code look "tall".  I am going to change the CMake files afffected by **this PR** to do that.

I have created #506 to automate all of this, we should not spend time thinking whether the file is formatted right or not (we can spend time thinking about whether the tool to format is configured right or not).